src/components/Datepicker/Datepicker.js

```diff
@@ -446,6 +445,16 @@ export class DatepickerRaw extends Component {
         )}
       </div>
     );
+
+    if (closeOnClickOutside) {
```

Comment:
I'm not sure whether this is approach scales well. Would it make sense to add a `active` prop to `<ClickOutside />`? That way, we would avoid conditional render statements in every feature that uses this